### PR TITLE
feat: make resources configurable by deployment

### PR DESCRIPTION
**Pull request by SteveHNH**
To optimize resources easily, these values should be made configurable using parameters.

I copied the current settings into the default values in the params so there's no need to change any params in app-interface.

JIRA: https://issues.redhat.com/browse/RHINENG-2174

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: clowdapp.yaml [hunk 1]
```diff
@@ -369,6 +369,66 @@ parameters:
...
+    value: 300m
+  - name: POPULATOR_MEMORY_LIMIT
+    description: memory limit for service
+    value: 1Gi
+  - name: POPULATOR_CPU_REQUESTS
+    description: cpu requests for service
+    value: 50m
+  - name: POPULATOR_MEMORY_REQUESTS
+    description: memory requests for service
+    value: 512mi
```
Comment (petrblaho) [inline clowdapp.yaml:383]: I guess this should be `512Mi`. See `The ClowdApp "historical-system-profiles" is invalid: spec.jobs[0].podSpec.resources.requests.memory: Invalid value: "512mi": spec.jobs[0].podSpec.resources.requests.memory in body should match '^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$'`in https://ci.ext.devshift.net/job/RedHatInsights-historical-system-profiles-backend-pr-check/580/console

**Review by petrblaho — CHANGES_REQUESTED** [review]
Looks good.

There's a need to change that one `m` to `M`.

Also, can you, please, link a ticket for this?

Thank you

**Comment by SteveHNH** [comment]
Added a JIRA link and fixed the typo in the template

File: clowdapp.yaml [hunk 2]
```diff
@@ -369,6 +369,66 @@ parameters:
   - name: IMAGE
     description: image name
     value: quay.io/cloudservices/historical-system-profiles-backend
+  - name: POPULATOR_CPU_LIMIT
+    description: cpu limit for service
+    value: 300m
+  - name: POPULATOR_MEMORY_LIMIT
```
[the remark this answers is not in the document]
Comment (petrblaho) [reply clowdapp.yaml:375]: Tracking here https://issues.redhat.com/browse/RHINENG-2178.

Let's get this merged now.

**Review by petrblaho — APPROVED** [review]

**Pull request merged**